data/building-hours/2-post-office.yaml

```diff
@@ -5,7 +5,11 @@ category: Sayles-Hill Campus Center
 schedule:
   - title: Hours
     hours:
-      - {days: [Mo, Tu, We, Th, Fr], from: '10:00am', to: '4:00pm'}
+      - {days: [Mo], from: '8:00am', to: '5:00pm'}
+      - {days: [Tu], from: '8:00am', to: '5:00pm'}
+      - {days: [We], from: '8:00am', to: '6:00pm'}
+      - {days: [Th], from: '8:00am', to: '6:00pm'}
+      - {days: [Fr], from: '8:00am', to: '5:00pm'}
```

Comment:
I broke these into one-per-day because otherwise we'd just have `Mo,Tu,Fr` and `We,Th` listed in the app… and that seems harder to parse than one-day-per-line.